src/main/resources/credentials.json

```diff
@@ -0,0 +1 @@
+{"web":{"client_id":"119851197452-crnk45b5i6gsi5povitstfpd203n7j6b.apps.googleusercontent.com","project_id":"course-scheduler-step-2020","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"cCGPOzzQ4-kMNwut3nbr-ztU","redirect_uris":["http://localhost:8080","http://localhost:8080/Callback","http://localhost:8888","http://localhost:8888/Callback"],"javascript_origins":["http://localhost:8080","http://localhost:8888"]}}
```

Comment:
FYI you would typically not want to check in a private key into a repo

https://developers.google.com/maps/api-key-best-practices

For the purposes of this project we can gloss over this but it's good to know when it comes to situations where having your key get compromised could lead to problems down the road, eg. if someone takes your key and uses it in all sorts of ways such that your paid account gets charged for that usage.